omc3/optics_measurements/coupling.py

```diff
@@ -217,19 +217,34 @@ def compensate_rdts_ryoichi():
 
 # ----- Helpers ----- #
 
-# def _take_next(phases: tfs.TfsDataFrame, shift: int = 1):
-#     """
-#     Takes the following BPM for momentum reconstruction by a given shift.
-#
-#     Args:
-#         phases (tfs.TfsDataFrame): Dataframe matrix of phase advances, as calculated in phase.py.
-#         shift (int): ???
-#     """
-#     indices = np.roll(np.arange(phases.to_numpy().shape[0]), shift)
-#     return indices, phases.to_numpy()[np.arange(phases.to_numpy().shape[0]), indices] - 0.25
+def _find_pair(phases: tfs.TfsDataFrame, mode: int = 1):
```

Comment:
Here `mode` defaults to 1 but at the command line it defauts to 0. We should make a choice

Reply:
mhh, good point. maybe get rid of the default altogether?

Reply:
No, we should just have the same default everywhere.

Reply:
well, given the fact that `_find_pair` is _always_ called with the second arg, as in ` bpm_pairs_y, deltas_y = _find_pair(phases_y, meas_input.coupling_pairing)` and it's a module 'private' function, I think we can safely get rid of the default. 
or put it to 0

Reply:
I am also fine with either so just make a decision so we can merge.

Reply:
Done in [d210ac9](https://github.com/pylhc/omc3/pull/371/commits/d210ac978961c23eb74577daf96d6883bfe28aaa)